expression/builtin_time.go

```diff
@@ -20,6 +20,7 @@ package expression
 import (
 	"context"
 	"fmt"
+	"github.com/pingcap/parser/ast"
```

Comment:
Please follow the existing style to put the import file.

Reply:
Oh, mistake. I will fix it